qa/rpc-tests/test_framework/mininode.py

```diff
@@ -1054,14 +1052,14 @@ def serialize(self):
         return r
 
     def __repr__(self):
-        return 'msg_version(nVersion=%i nServices=%i nTime=%s addrTo=%r addrFrom=%r nNonce=0x%016X strSubVer=%s nStartingHeight=%i)' \
+        return 'msg_version(nVersion=%i nServices=%i nTime=%s addrTo=%s addrFrom=%s nNonce=0x%016X strSubVer=%s nStartingHeight=%i)' \
```

Comment:
Is this change intentional?

Reply:
Will revert. I made a mistake melding.

Reply:
Nope. This was a mistake with meld.

Reply:
It seems not to have been reverted though.